symplyphysics/laws/thermodynamics/internal_energy_change_via_amount_of_heat_and_work_done.py

```diff
@@ -0,0 +1,60 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    # symbols,
+)
+
+# Description
+## The first law of thermodynamics is a formulation of the law of conservation of energy in the
+## context of thermodynamic processes. It relates the change in internal energe of the system,
+## work done on or by the system and the amount of heat supplied to or widthrawn from the system
+## during a thermodynamic process.
+
+# Law: dU = delta Q - delta W
+## dU - infinitesimal change in internal energy of system
+## delta Q - inexact differential of an infinitesimal amount of heat supplied to
+##           or withdrawn from the system during the interaction with its surroundings
+## delta W - inexact differential of infinitesimal work done by the system
+
+# Note
+## - Work done _by_ the system is negative, work done _on_ the system by the environment is positive.
+## - Heat supplied _to_ the system is positive, heat lost _by_ the system is negative.
+## - This formula can be extended to finite processes, with the only change being that the
+##   changes are not infinitesimal but finite (`dU` would refer to a finite change in the system's
+##   internal energy, delta Q would be the total amount of heat supplied and delta W would be the total
+##   work done by the system).
+
+# Conditions
+## - This formula applies to any infinitesimal thermodynamic process, be it (quasi)static or not.
+
+internal_energy_change = Symbol("internal_energy_change", units.energy)
+heat_supplied_to_system = Symbol("heat_supplied_to_system", units.energy)
+work_done_by_system = Symbol("work_done_by_system", units.energy)
+
+law = Eq(internal_energy_change, heat_supplied_to_system - work_done_by_system)
```

Comment:
I think that with some amount of heat supplied to the system and same amount of work done by the system, internal energy should be zero. But with your signs convention it amounts to double amount of heat supplied.